src/createBrowserApp.js

```diff
@@ -112,7 +119,10 @@ export default function createBrowserApp(App) {
           })
         );
       if (newState && newState !== lastState) {
-        this.setState({ nav: newState }, dispatchEvents);
+        this.setState({ nav: newState }, () => {
+          this._onNavigationStateChange(lastState, newState, action);
+          dispatchEvents();
```

Comment:
Same order as the native app container, nice 👌

Reply:
we have Gitter, slack, or somethings like this for our community? 
actually i start to fix some major problem on my top head and would like to avoid conflict with other contributer